### ISSUE TITLE
Puppeteer does not capture gifs properly

### ISSUE THREAD

**Issue by avaer** [webaverse/xrpackage#112]
It looks like on MacOS, puppeteer makes empty gifs of this package: http://packages.exokit.org/animu

The screenshot: https://ipfs.exokit.org/ipfs/QmbmkBHPzWDh17gUA5Q7stpKnt7y4tmUvA27mzaHLqRv4K.gif

It might have regressed with https://github.com/webaverse/xrpackage/pull/101 ?

**Comment by shu8** [comment]
I tried running this on a linux instance and it seems to fail on that too. The exact error/warnings are like:

```
WebGL: INVALID_OPERATION: renderbufferStorageMultisample: samples out of range
...
[.WebGL-0x1f6eefec4e00]GL ERROR :GL_INVALID_FRAMEBUFFER_OPERATION : glDrawElements: framebuffer incomplete
```

![puppeteer-bake-error](https://user-images.githubusercontent.com/8850830/89196109-a8ffaa80-d5a1-11ea-8333-cc23f74d6a9f.png)

and then the page crashes.

It works fine on my Windows machine though! 

Do you have an thoughts on where to start debugging for this error?

**Comment by avaer** [comment]
@shu8 Yes, the first step is to find which piece of code is causing this difference. Run this in a non-puppeteer browser. Also, run the package without xrpk wrapping the WebGL calls, in a non-puppeteer browser.

This could be caused by the fact that we try to create a multisampled framebuffer for AA, but this linux box/VM doesn't support it: https://github.com/webaverse/xrpackage/blob/master/xrpackage.js#L482

**Comment by avaer** [comment]
https://github.com/webaverse/xrpackage/issues/114
